zone/trading.cpp

```diff
@@ -1151,8 +1152,8 @@ void Client::Trader_EndTrader() {
 			}
 
 			safe_delete(outapp);
-			safe_delete(gis);
 		}
+		safe_delete(gis);
```

Comment:
Just undo this change and push a new commit to this pr's branch to resolve the #2266 duplicate

Reply:
Done